.travis.yml

```diff
@@ -72,6 +73,22 @@ matrix:
     - env: BUILD_TYPE="Debug" WITH_SYMENGINE_ASSERT="yes" WITH_PYTHON="yes" PYTHON_VERSION="2.7"
       compiler: gcc
       os: osx
+    - language: ruby
+      cache: bundler
+      env: TEST_IN_TREE="yes" WITH_RUBY="yes"
+      rvm: 1.9.3
```

Comment:
You aren't suggesting a matrix, right? That would be total of 8 jobs, instead of 4 now. Like the first two with `gcc` and the second two with `clang`?